### DISCUSSION TITLE
Integrating unsupported blockchain - sendCustomRequest in v2

### DISCUSSION THREAD

**Discussion by mzibara** [WalletConnect/walletconnect-monorepo#2591] [Developer Support] [answered]
Hey! I'm working with internet computer blockchain, that is not yet supported by wallet connect officially. With v1 we used workaround calling sendCustomRequest,  is there still some workaround in v2,  or ability to scope using custom requiredNamespaces. Thnx in advance 🙂

**Comment by finessevanes** [comment, marked as answer]
@mzibara was this method approved during the pairing process? In v2, you can only send requests with methods that are in `session.namespaces`.

**Reply by mzibara** [reply]
Hey @finessevanes yes that is what I've also figured out. Thank you for the reply :)

**Reply by benduongquangmobile** [reply]
could you guys take an example?
